src/AppController.m

```diff
@@ -1014,11 +1007,15 @@ -(void)setLayout:(NSInteger)newLayout withRefresh:(BOOL)refreshFlag
     if (refreshFlag)
         [articleController.mainArticleView refreshFolder:MA_Refresh_RedrawList];
 	[browserView setPrimaryTabItemView:articleController.mainArticleView];
+	foldersTree.mainView.nextKeyView = [browserView primaryTabItemView].mainView;
+    if (self.selectedArticle == nil)
```

Comment:
I should have gone the `[mainWindow makeFirstResponder:(self.selectedArticle == nil) ?  foldersTree.mainView : [browserView primaryTabItemView].mainView];` way 😈